<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewLink</class>
 <widget class="QDialog" name="AddNewLink">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="BookmarkValueLineEdit" name="leLink"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="BookmarkValueLineEdit" name="leName"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tags</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="BookmarkValueLineEdit" name="leTags"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="BookmarkComboBox" name="cbFolders"/>
     </item>
     <item row="3" column="1">
      <widget class="ComplexLineEdit" name="widget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BookmarkComboBox</class>
   <extends>QComboBox</extends>
   <header>bookmarkcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>BookmarkValueLineEdit</class>
   <extends>QLineEdit</extends>
   <header>bookmarkvaluelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComplexLineEdit</class>
   <extends>QWidget</extends>
   <header>complexlineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leLink</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>leTags</tabstop>
  <tabstop>cbFolders</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddNewLink</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddNewLink</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
